### PR TITLE
JW-209 Mocked bridge and nexthop requests.

### PR DESCRIPTION
**Pull request by Attil**

### CODE REVIEW

File: windows/vr_host.c [hunk 1]
```diff
@@ -6,6 +6,8 @@
 #include "vr_windows.h"
 #include "vrouter.h"
 
+extern void vif_attach(struct vr_interface *);
```
Comment (michal-clapinski) [inline windows/vr_host.c]: Maybe you should create a header for this externs?

File: windows/vrouter_mod.c [hunk 2]
```diff
@@ -5,6 +5,8 @@
 #include "vr_packet.h"
 #include "vr_sandesh.h"
 
+extern void vif_attach(struct vr_interface *);
```
Comment (michal-clapinski) [inline windows/vrouter_mod.c]: Maybe you should put this in a header?

File: windows/vrouter_mod.c [hunk 3]
```diff
@@ -967,18 +1001,25 @@ SxExtStartCompleteNetBufferListsIngress(
         SxLibCompleteNetBufferListsIngress(Switch,
             NetBufferLists,
             SendCompleteFlags);
+        DbgPrint("Done!\r\n");
```
Comment (michal-clapinski) [inline windows/vrouter_mod.c]: Maybe add some prefix to this Print so we would know what is done.

File: windows/vrouter_mod.c [hunk 4]
```diff
@@ -967,18 +1001,25 @@ SxExtStartCompleteNetBufferListsIngress(
...
     }
 
+
     PNET_BUFFER_LIST curNbl = NetBufferLists;
     PNET_BUFFER_LIST nextNbl = NULL;
 
     while (curNbl != NULL)
     {
+        DbgPrint("Looping...\r\n");
```
Comment (michal-clapinski) [inline windows/vrouter_mod.c]: Previous comment applies also here.

File: windows/vrouter_mod.c [hunk 5]
```diff
@@ -967,18 +1001,25 @@ SxExtStartCompleteNetBufferListsIngress(
...
         struct vr_packet* pkt = win_get_packet_from_nbl(curNbl);
         if (pkt != NULL)
+        {
+            pkt->vp_net_buffer_list = NULL;
             windows_host.hos_pfree(pkt, VP_DROP_DISCARD);
+        }
 
         curNbl = nextNbl;
     }
+    DbgPrint("After looping\r\n");
```
Comment (michal-clapinski) [inline windows/vrouter_mod.c]: Previous comment applies also here.

File: windows/vrouter_mod.c [hunk 6]
```diff
@@ -152,6 +154,38 @@ AddNicToArray(struct vr_switch_context* ctx, struct vr_nic* nic, NDIS_IF_COUNTED
         }
     }
 
+    // TODO: REMOVE THIS
```
Comment (michal-clapinski) [inline windows/vrouter_mod.c]: Could you put a comment here explaining why you put this code here and what it does?

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by m-kostrzewa** [comment]
Retest this please

**Comment by Attil** [comment]
After some thinking with @sodar, please don't merge this. We've decided we should not merge mocks into the main branch. I'll keep this with parity to windows branch as needed.

**Pull request closed without merging**